src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -388,4 +388,40 @@ Error({Kind:ErrorKind.BadLanguageCode})
 "2023-12-21T12:34:56.789Z"
 
 >> Text(DateTimeValue("12:34:56"),"yyyy-mm-dd hh:mm:ss")
-"1899-12-30 12:34:56"
\ No newline at end of file
+"1899-12-30 12:34:56"
+
+>> Text(Value("1,234","en-US"),"[$-fr-FR]#,###","fr-FR")
```

Comment:
>Text(Value("1,234","en-US"),"[$-fr-FR]#,###","fr-FR") [](http://example.com/codeflow?start=3&length=53)

Those tests don't need to use the Value function - we can use a constant number since we're testing the Text function here. A good number to test would be 1234.5678 - it will have both decimal separator and possible thousands separator as well.

Some good tests:
    >> Text(1234.5678, "[$-en-US]#,##0.00", "en-US")
    "1,234.57"
    
    >> Text(1234.5678, "[$-pt-BR]#.##0,00", "en-US")
    "1,234.57"
    
    >> Text(1234.5678, "[$-pt-BR]#.##0,00", "fr-FR")
    "1 234,57"
    
    >> Text(1234.5678, "[$-en-US]#,##0.00", "pt-BR")
    "1.234,57"
    
    // If not closed by a ']', then not interpreted as a format specifier
    >> Text(1234.5678, "[$-en-US#,##0.00", "en-US")
    "[$-en-US1,234.57"
    
    >> Text(1234.5678, "[$-fr]# ##0,00", "en-US")
    "1,234.57"
    
    >> Text(1234.5678, "[$-fr-FR]# ##0,00", "vi-VI")
    "1.234,57"
    
    >> Text(1234.5678, "[$-en-US]#,##0.00", "en-US")
    "1,234.57"
    
    >> Text(1234.5678, "[$-pt-BR]#.##0,00", "en-US")
    "1,234.57"
    
    >> Text(1234.5678, "[$-pt-BR]#.##0,00", "fr-FR")
    "1 234,57"
    
    >> Text(1234.5678, "[$-en-US]#,##0.00", "pt-BR")
    "1.234,57"
    
    // If not closed by a ']', then not interpreted as a format specifier
    >> Text(1234.5678, "[$-en-US#,##0.00", "en-US")
    "[$-en-US1,234.57"
    
    >> Text(1234.5678, "[$-fr]# ##0,00", "en-US")
    "1,234.57"
    
    // Unknown locales would default to en-US in C# (error in Power Apps)
    >> Text(1234.5678, "[$-xx-YY]0,0", "en-US")
    "1,235"

Reply:
Can you include the test comments too?

Reply:
update

Reply:
are these all added?
I don't see the last one